dll/win32/shell32/shlfileop.cpp

```diff
@@ -1972,14 +1974,26 @@ validate_operation(LPSHFILEOPSTRUCTW lpFileOp, FILE_LIST *flFrom, FILE_LIST *flT
                     {
                         strTitle.LoadStringW(IDS_MOVEERRORTITLE);
                         if (IsAttribDir(feFrom->attributes))
+#ifdef __REACTOS
+                            strText.Format(IDS_MOVEERRORSAMEFOLDER, GET_FILENAME(feFrom));
+#else
                             strText.Format(IDS_MOVEERRORSAMEFOLDER, feFrom->szFilename);
```

Comment:
I very much doubt the filename is empty in this case and the one directly below (l1986)

Reply:
KRosUser's test says that the string can be empty.